src/analyses/libraryFunctions.ml

```diff
@@ -29,6 +37,34 @@ let posix_descs_list: (string * LibraryDesc.t) list = LibraryDsl.[
     ("__builtin_bzero", special [__ "dest" [w]; __ "count" []] @@ fun dest count -> Bzero { dest; count; });
     ("explicit_bzero", special [__ "dest" [w]; __ "count" []] @@ fun dest count -> Bzero { dest; count; });
     ("__explicit_bzero_chk", special [__ "dest" [w]; __ "count" []; drop "os" []] @@ fun dest count -> Bzero { dest; count; });
+    ("nl_langinfo", unknown [drop "item" []]);
+    ("nl_langinfo_l", unknown [drop "item" []; drop "locale" [r]]);
```

Comment:
At least for glibc, it is a pointer to a struct, see the `__locale_t` [here](https://github.com/lattera/glibc/blob/master/locale/bits/types/__locale_t.h).

Reply:
Thanks for looking that up! Since the struct contains other pointers that the implementation probably uses in some way, `r_deep` would be the safe thing to do.